toolkit/offchain/src/csl.rs

```diff
@@ -381,7 +385,7 @@ impl TransactionBuilderExt for TransactionBuilder {
 	}
 }
 
-pub(crate) trait InputsBuilderExt {
+pub(crate) trait InputsBuilderExt: Sized {
```

Comment:
Why does it have to be Sized?

Reply:
to be used in`Result`. `InputsBuilder` is `Sized` luckily